packages/zipkin-instrumentation-express/test/wrapExpressHttpProxyIntegrationTest.js

```diff
@@ -221,31 +221,35 @@ describe('express http proxy instrumentation - integration test', () => {
 
               expect(annotations[4].annotation.annotationType).to.equal('LocalAddr');
 
-              expect(annotations[5].annotation.annotationType).to.equal('ServiceName');
-              expect(annotations[5].annotation.serviceName).to.equal('weather-app');
+              expect(annotations[5].annotation.annotationType).to.equal('BinaryAnnotation');
```

Comment:
hmm I don't think we want to create pseudo-annotations for debug, do we? We already propagate the trace span/trace ID, which is use in conversion like so: https://github.com/openzipkin/zipkin-js/blob/master/packages/zipkin/src/model.js#L43

I don't really like creating pseudo tags anyway unless absolutely required as it is easy for them to be double-added